modules/tinymce/src/core/test/ts/browser/content/InsertUnsanitizedContentTest.ts

```diff
@@ -0,0 +1,40 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.content.InsertUnsanitizedContentTest', () => {
+  const unsanitizedHtml = '<p><a href="javascript:alert(1)">XSS</a></p>';
+  const sanitizedHtml = '<p><a>XSS</a></p>';
+  const testInsertContent = (editor: Editor, content: string, expected: string) => {
+    const initialContent = '<p>initial</p>';
+    editor.setContent(initialContent);
+    TinySelections.setCursor(editor, [ 0, 0 ], 0);
+    editor.insertContent(content);
+    TinyAssertions.assertContent(editor, `${expected}\n${initialContent}`);
+  };
+
+  context('TINY-9600: Inserting unsanitized html with xss_sanitization: true', () => {
+    const hook = TinyHooks.bddSetupLight<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      xss_sanitization: true
```

Comment:
```suggestion
```
Since this is `true` by default?

Reply:
Good point, it's useful to have the test ensure that `xss_sanitization` is actually set to true by default.

Reply:
@tiny-ben-tran On second thoughts, not setting `xss_sanitization` would have the same effect as existing tests that test editor content sanitization. This test is here to ensure that the new option works as it is intended, so that we don't have logic that disables sanitization when `xss_sanitization` is set to `true` as well.